src/stat/corcov.ts

```diff
@@ -0,0 +1,16 @@
+import { Tensor, sub, mean, transpose, matMul, divNoNan, RecursiveArray } from '@tensorflow/tfjs-core';
```

Comment:
```suggestion
import { Tensor, RecursiveArray, sub, mean, transpose, matMul, divNoNan } from '@tensorflow/tfjs-core';
```

Would you please split these by type and method?